### PR TITLE
Checkout rensonsive widget style fix

### PR DESCRIPTION
**Pull request by pronto2000**
Fixed an issue with responsive CSS styles which caused widgets to overlap.

### CODE REVIEW

File: skin/frontend/base/default/amazon_payments/css/styles_checkout.css [hunk 1]
```diff
@@ -185,19 +185,14 @@ body.page-popup #amazon-checkout {
 
 textarea#sandbox { width:100%;}
 
-
-/* Responsive */
-@media all and (max-width: 880px) {
-  #checkout-amazon-columns,
+@media all and (max-width: 767px) {
```
Comment (islandskater43) [inline skin/frontend/base/default/amazon_payments/css/styles_checkout.css]: @pronto2000 - why the change from 880px to 767px ?

File: skin/frontend/base/default/amazon_payments/css/styles_checkout.css [hunk 2]
```diff
@@ -185,19 +185,14 @@ body.page-popup #amazon-checkout {
...
+      display: block;
   }
-
-  .col-amz {
-    display:block;
-    width:auto;
-    margin-right:0;
-    padding-right:0;
-    border-right:0;
+  #col-1-amz,
```
Comment (islandskater43) [inline skin/frontend/base/default/amazon_payments/css/styles_checkout.css]: Did the col-x-amz IDs already exist?  And are we sure that it aligns perfectly with the .col-amz that was previously declared?

Comment (pronto2000) [reply skin/frontend/base/default/amazon_payments/css/styles_checkout.css]: Yes and yes.

Here's the output of _grep -r col-.-amz ._:
`./app/design/frontend/base/default/template/amazon_payments/checkout.phtml:            <div id="col-1-amz" class="col-amz">
./app/design/frontend/base/default/template/amazon_payments/checkout.phtml:            <div id="col-2-amz" class="col-amz">
./app/design/frontend/base/default/template/amazon_payments/checkout.phtml:            <div id="col-3-amz" class="col-amz">`

File: skin/frontend/base/default/amazon_payments/css/styles_checkout.css (hunk 1 above)
Comment (pronto2000) [reply skin/frontend/base/default/amazon_payments/css/styles_checkout.css]: Mostly, because 768px is the breakpoint majority of frontend developers use to switch between desktop and mobile. For example M2 documentation is very clear about that:

http://devdocs.magento.com/guides/v2.2/frontend-dev-guide/responsive-web-design/rwd_css.html

M1 default rwd theme not that explicit, however most third party themes still use this approach. This is the reason why switching to width: 100% as early as 880px didn't make sense.
